support/ci/lint.sh

```diff
@@ -28,6 +28,7 @@ warn() {
       >&2 echo -e "   \033[1;32m${program}: \033[1;33mWARN \033[1;37m$1\033[0m"
       ;;
     *)
+      # shellcheck disable=2154
```

Comment:
```
In ./support/ci/lint.sh line 31:
      >&2 echo "   ${pkg_name}: WARN $1"
                   ^-- SC2154: pkg_name is referenced but not assigned.
```
Should this be
```bash
       >&2 echo "   ${program}: WARN $1"
```
to match the other arm of the conditional?

Reply:
It certainly looks that way to me.